www/source/docs/using-builder.html.md.erb

```diff
@@ -339,6 +675,10 @@ You can upload keys for the origin through the web interface for Builder, or by
 
 ### Upload Packages to Builder
 
+The purpose of Chef Habitat Builder is
```

Comment:
Seems to me an unfinished thought perhaps?

Reply:
An unfinished thought! Thanks for finding it!